### ISSUE TITLE
[1.11+] Hyperlink tooltips don't work any longer

### ISSUE THREAD

**Issue by DHowett** [microsoft/terminal#11735]
### Windows Terminal version (or Windows build number)

1.11, 1.12, 1.13+

### Steps to reproduce

powershell:

```
"`e]8;;https://howett.net`e\Link Text Here`e]8;;`e\"
```

### Expected Behavior

Tooltip on hover

### Actual Behavior

Nothing! Did Control/ControlInteractivity/ControlCore break this?

**Comment by elsaco** [comment]
@DHowett isn't that escape code valid in PS7 only? Does not work with PS6 or older. 

Sample PS7 screenshot:
![pwsh_link_test](https://user-images.githubusercontent.com/3933920/141348494-56f68483-7443-41d5-93c9-aad4e3e2b53e.png)

**Comment by DHowett** [comment]
Yes, it is valid in the only versions of PowerShell that matter.

Interesting that it works for you! Maybe this is a Windows build issue instead of a Terminal issue...

**Comment by zadjii-msft** [comment]
![image](https://user-images.githubusercontent.com/18356694/141350826-5a0d2633-bad3-4695-b905-11e55342394a.png)

I bet it's a DPI issue

**Comment by zb140** [comment]
One more random data point in case someone finds it useful: I seem to be in an in-between world where sometimes it works and sometimes it doesn't.  It seems that if I move the mouse in from the left side of the screen on the same line as the link, I usually get no tooltip. If the mouse comes from the bottom (or otherwise goes over _non_-link text) before hitting the link, I do get the tooltip.  Even coming from the left, I still get it maybe 10% of the time (estimated).

**Comment by DHowett** [comment]
Huh, you're right. If I keep re-aiming and trying again, it works. I'm going to repurpose this bug...

**Comment by PankajBhojwani** [comment]
So the reason for this is that the invisible border we create to 'house' the tooltip is only the size of 1 cell, so unless your cursor is exactly on that 1 cell you won't get the tooltip. Here's a more detailed overview of the process:

1. Cursor moves to the link
2. We immediately create the 1-cell size border with the tooltip, and place the border on the cell the cursor is on
3. Cursor moves to another part of the link (without moving 'out' of the link) - we don't move the border again because the cursor has remained on the same link
4. No tooltip because the cursor isn't on that exact cell with the border

Possible solutions would be

- Make the border cover the entire link
    - This would probably be the ideal solution, but implementation wise is clunky because links that span multiple lines will need multiple borders
- Move the 1-cell size border even when the cursor is moving within the same link

**Comment by DHowett** [comment]
Hmmm. Maybe another possible solution would be to switch to manual tooltip presentation like @zadjii-msft implemented in #11680.
